src/translate.py

```diff
@@ -1957,8 +1957,11 @@ class RegInfo:
     stack_info: StackInfo = field(repr=False)
     contents: Dict[Register, RegData] = field(default_factory=dict)
     read_inherited: Set[Register] = field(default_factory=set)
+    active_instr: Optional[Instruction] = None
```

Comment:
I don't *love* this... another option is to keep track of just the list of `inputs`/`outputs`? (That way no one can use this field to do `regs.active_instr.mnemonic` or other oddities?)

Reply:
underscore-prefix it, maybe?